tikv/go-client-api.md

```diff
@@ -17,7 +17,9 @@ To use the Raw Key-Value API in applications developed by golang, take the follo
 1. Install the necessary packages.
 
     ```bash
-    go get -v -u github.com/pingcap/tidb/store/tikv
+    export GO111MODULE=on
+    go mod init rawkv-demo
+    go get github.com/pingcap/tidb@master
```

Comment:
I'm not sure go get master could work. Did you try it?

Reply:
It works. I tested it.